src/prefect/tasks/jupyter/jupyter.py

```diff
@@ -0,0 +1,66 @@
+import nbconvert
+import nbformat
+import papermill as pm
+
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class JupyterTask(Task):
```

Comment:
We're moving away from tasks with `Task` suffixes (which feel redundant). Perhaps `RunNotebook` or `ExecuteNotebook` instead?